README.md

```diff
@@ -8,45 +8,58 @@
 component) or standalone (single component) modes.  </strong>
 </p>
 
-[![Downloads](https://img.shields.io/badge/dynamic/json?url=http%3A%2F%2Fjuliapkgstats.com%2Fapi%2Fv1%2Ftotal_downloads%2FClimaLand&query=total_requests&suffix=%2Ftotal&label=Downloads)](http://juliapkgstats.com/pkg/ClimaLand)
+<div align="center">
 
-Recommended Julia Version: Stable release v1.10.0. CI no longer tests earlier
-versions of Julia.
+|||
+|---------------------:|:----------------------------------------------|
+| **Docs Build**       | [![docs build][docs-bld-img]][docs-bld-url]   |
+| **Documentation**    | [![dev][docs-dev-img]][docs-dev-url]          |
+| **GHA CI**           | [![gha ci][gha-ci-img]][gha-ci-url]           |
+| **Code Coverage**    | [![codecov][codecov-img]][codecov-url]        |
+| **Downloads**        | [![downloads][downloads-img]][downloads-url]  |
 
-Certain features, including global runs, are not currently available on
-Windows due to limitations with our regridding software.
+</div>
 
-Note that ClimaLand.jl is a different model from the original CliMA Land,
-which aims to utilize remote sensing data through more complex canopy RT
-and plant physiology modules. For more details, please refer to
-https://github.com/CliMA/Land.
-- Wang, Yujie, et al. "Testing stomatal models at the stand level in deciduous angiosperm and evergreen gymnosperm forests using CliMA Land (v0. 1)." Geoscientific Model Development 14.11 (2021): 6741-6763.
-- R. K. Braghiere, Y. Wang, R. Doughty, D. Souza, T. Magney, J. Widlowski, M. Longo, A. Bloom, J. Worden, P. Gentine, and C. Frankenberg. 2021. Accounting for canopy structure improves hyperspectral radiative transfer and sun-induced chlorophyll fluorescence representations in a new generation Earth System model. Remote Sensing of Environment. 261: 112497.
-- Wang, Yujie, and Christian Frankenberg. "On the impact of canopy model complexity on simulated carbon, water, and solar-induced chlorophyll fluorescence fluxes." Biogeosciences 19.1 (2022): 29-45.
-- Wang, Yujie, et al. "GriddingMachine, a database and software for Earth system modeling at global and regional scales." Scientific data 9.1 (2022): 258.
-- Holtzman, Nataniel, et al. "Constraining plant hydraulics with microwave radiometry in a land surface model: Impacts of temporal resolution." Water Resources Research 59.11 (2023): e2023WR035481.
+# ClimaLand.jl
+
+This is the repository of the CliMA land model code. Here are some notable features:
+- ClimaLand has a modular design, models can be run as standalone (e.g., soil moisture only) or integrated (e.g., soil moisture and energy AND canopy AND snow, etc.)
+- ClimaLand is CPU and GPU compatible
+- ClimaLand welcome contributions: we welcome users to create a branch and extend our code. For example, a modeler might want to test a new stomatal conductance model. 
+- ClimaLand provides APIs and UIs at multiple levels. 
```

Comment:
what do you mean by multiple levels?

Reply:
You're right this isn't clear... I meant for example ClimaLand and ClimaLandSimulations API, where you need to go in details or not to run a simulation, for example. (I think ClimaLandSimulations is "higher level" because it does a lot of abstraction? Is this correct language to say this? I am not sure)

Should I just say "ClimaLand provides APIs and UIs?"

Reply:
hmm you could say we provide APIs for model developers, as well as a higher-level user interface for running simulations?